### ISSUE TITLE
Multiple `notation` error on Nuxt 2 

### ISSUE THREAD

**Issue by serkon** [stylelint/stylelint#7217]
### What minimal example or steps are needed to reproduce the bug?

When I open my Nuxt project in VSCode, I see many error messages when hovering over style elements. However, I don't see any errors in the console. Even though VSCode displays many warnings related to notation errors, the application works as expected.

**Sample: 1**
![Screenshot 2023-10-07 202801](https://github.com/stylelint/stylelint/assets/71252/091c92b0-3cfd-49c5-832d-caac016eaf60)
**Sample: 2**
![Screenshot 2023-10-07 204055](https://github.com/stylelint/stylelint/assets/71252/c9b030c3-88e0-4065-b684-55adc6467bce)

**Sample: 3**
![Screenshot 2023-10-07 204148](https://github.com/stylelint/stylelint/assets/71252/abacf698-fb75-4070-8cac-84646ee7383b)

```scss
@import 'variables';

body {
  background-color: $primary-color;
}
```

### What minimal configuration is needed to reproduce the bug?

```js
module.exports = {
  plugins: ['stylelint-no-unsupported-browser-features', 'stylelint-order'],
  customSyntax: 'postcss-scss',
  extends: ['stylelint-config-standard', 'stylelint-config-recommended-vue'],
  // add your custom config here
  // https://stylelint.io/user-guide/configuration
  rules: {
    'color-hex-length': 'long',
    'declaration-block-no-redundant-longhand-properties': [
      true,
      {
        ignoreShorthands: ['flex-flow'],
      },
    ],
    'declaration-block-no-duplicate-properties': null,
    'at-rule-empty-line-before': [
      'always',
      {
        except: ['after-same-name'],
      },
    ],
    'property-no-unknown': [
      true,
      {
        ignoreProperties: ['composes', 'r'],
      },
    ],
    'unit-no-unknown': [
      true,
      {
        ignoreUnits: ['x'],
      },
    ],
    'order/order': [
      ['custom-properties', 'at-variables', 'declarations', 'at-rules', 'rules', 'less-mixins'],
      {
        severity: 'warning',
      },
    ],
    'order/properties-order': [
      [
        {
          emptyLineBefore: 'never',
          properties: ['content'],
        },
        {
          emptyLineBefore: 'never',
          properties: ['position', 'top', 'right', 'bottom', 'left', 'z-index'],
        },
        {
          emptyLineBefore: 'never',
          properties: [
            'box-sizing',
            'display',
            'flex',
            'flex-basis',
            'flex-direction',
            'flex-flow',
            'flex-grow',
            'flex-shrink',
            'flex-wrap',
            'align-content',
            'align-items',
            'align-self',
            'justify-content',
            'order',
            'margin',
            'margin-top',
            'margin-right',
            'margin-bottom',
            'margin-left',
            'padding',
            'padding-top',
            'padding-right',
            'padding-bottom',
            'padding-left',
            'min-width',
            'min-height',
            'max-width',
            'max-height',
            'width',
            'height',
            'float',
            'clear',
            'clip',
            'visibility',
            'overflow',
            'border',
            'border-top',
            'border-right',
            'border-bottom',
            'border-left',
            'border-width',
            'border-top-width',
            'border-right-width',
            'border-bottom-width',
            'border-left-width',
            'border-style',
            'border-top-style',
            'border-right-style',
            'border-bottom-style',
            'border-left-style',
            'border-radius',
            'border-top-left-radius',
            'border-top-right-radius',
            'border-bottom-right-radius',
            'border-bottom-left-radius',
            'border-color',
            'border-top-color',
            'border-right-color',
            'border-bottom-color',
            'border-left-color',
            'box-shadow',
          ],
        },
        {
          emptyLineBefore: 'never',
          properties: [
            'background',
            'background-attachment',
            'background-clip',
            'background-color',
            'background-image',
            'background-repeat',
            'background-position',
            'background-size',
          ],
        },
        {
          emptyLineBefore: 'never',
          properties: [
            'color',
            'font',
            'font-family',
            'font-size',
            'font-smoothing',
            'font-style',
            'font-variant',
            'font-weight',
            'letter-spacing',
            'line-height',
            'list-style',
            'text-align',
            'text-decoration',
            'text-indent',
            'text-overflow',
            'text-rendering',
            'text-shadow',
            'text-transform',
            'text-wrap',
            'vertical-align',
            'white-space',
            'word-spacing',
          ],
        },
      ],
      {
        severity: 'warning',
      },
    ],
    'block-no-empty': true,
    'rule-empty-line-before': ['always', { except: ['after-single-line-comment', 'first-nested'] }],
    'comment-no-empty': true,
    'comment-empty-line-before': 'never',
    'declaration-empty-line-before': [
      'always',
      {
        except: ['after-comment', 'after-declaration', 'first-nested'],
      },
    ],
    'no-duplicate-selectors': true,
    'no-duplicate-at-import-rules': true,
    'no-invalid-position-at-import-rule': null,
    'color-function-notation': 'legacy',
    'import-notation': 'string',
    'at-rule-no-unknown': null,
  },
  overrides: [
    {
      files: ['*.vue', '**/*.vue'],
      customSyntax: 'postcss-html',
    },
    {
      files: ['components/**/*.css', 'pages/**/*.css'],
      rules: {
        'alpha-value-notation': 'percentage',
      },
    },
  ],
}
```

### How did you run Stylelint?

This is a nuxt 2 project.

```shell
npm run dev
```

### Which Stylelint-related dependencies are you using?

```json
{
  "name": "ts-starter",
  "version": "1.0.0",
  "private": true,
  "scripts": {
    "dev": "nuxt",
    "build": "nuxt build",
    "start": "nuxt start",
    "generate": "nuxt generate",
    "lint:js": "eslint --ext \".js,.ts,.vue,.json\" --ignore-path .gitignore .",
    "lint:style": "stylelint \"**/*.{css,scss,sass,html,vue}\" --ignore-path .gitignore",
    "lint:prettier": "prettier --check .",
    "lint": "npm run lint:js && npm run lint:style && npm run lint:prettier",
    "lintfix": "prettier --write --list-different . && npm run lint:js -- --fix && npm run lint:style -- --fix",
    "prepare": "husky install && npm run setup:husky",
    "setup:husky": "husky set .husky/pre-push \"npx lint-staged && npm test -- --watchAll=false\" && husky set .husky/commit-msg \"npx commitlint --edit $1\"",
    "test": "jest"
  },
  "husky": {
    "hooks": {
      "pre-commit": "lint-staged",
      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
    }
  },
  "lint-staged": {
    "*.{js,ts,vue}": "eslint --cache",
    "*.{css,scss,sass,html,vue}": "stylelint",
    "*.**": "prettier --check --ignore-unknown"
  },
  "dependencies": {
    "@nuxtjs/axios": "^5.13.6",
    "@nuxtjs/pwa": "^3.3.5",
    "core-js": "^3.25.3",
    "element-ui": "^2.15.10",
    "nuxt": "^2.17.0",
    "vue": "^2.7.10",
    "vue-server-renderer": "^2.7.10",
    "vue-template-compiler": "^2.7.10"
  },
  "devDependencies": {
    "@babel/eslint-parser": "^7.19.1",
    "@commitlint/cli": "^17.1.2",
    "@commitlint/config-conventional": "^17.1.0",
    "@nuxt/types": "^2.15.8",
    "@nuxt/typescript-build": "^2.1.0",
    "@nuxtjs/eslint-config-typescript": "^12.1.0",
    "@nuxtjs/eslint-module": "^3.1.0",
    "@nuxtjs/stylelint-module": "^4.1.0",
    "@nuxtjs/tailwindcss": "6.7.0",
    "@typescript-eslint/eslint-plugin": "^6.7.4",
    "@typescript-eslint/parser": "^6.7.4",
    "@vue/test-utils": "^1.3.0",
    "babel-core": "7.0.0-bridge.0",
    "babel-jest": "^29.1.2",
    "eslint": "^8.50.0",
    "eslint-config-prettier": "^8.5.0",
    "eslint-plugin-import": "^2.28.1",
    "eslint-plugin-jest": "^27.0.4",
    "eslint-plugin-json-format": "^2.0.1",
    "eslint-plugin-nuxt": "^4.0.0",
    "eslint-plugin-prettier": "^4.1.0",
    "eslint-plugin-simple-import-sort": "^10.0.0",
    "eslint-plugin-unused-imports": "^3.0.0",
    "eslint-plugin-vue": "^9.17.0",
    "husky": "^8.0.3",
    "jest": "^29.7.0",
    "jest-environment-jsdom": "^29.7.0",
    "jsonc-eslint-parser": "^2.3.0",
    "lint-staged": "^14.0.1",
    "postcss": "^8.4.31",
    "postcss-html": "^1.5.0",
    "postcss-scss": "^4.0.9",
    "prettier": "^2.7.1",
    "sass": "^1.69.0",
    "sass-loader": "^10.1.1",
    "stylelint": "^15.6.2",
    "stylelint-config-recommended-vue": "^1.4.0",
    "stylelint-config-standard": "^33.0.0",
    "stylelint-no-unsupported-browser-features": "^7.0.0",
    "stylelint-order": "^6.0.3",
    "ts-jest": "^29.0.3",
    "typescript": "^5.2.2",
    "vue-jest": "^3.0.4"
  }
}
```

### What did you expect to happen?
How could remove this errors on vscode?

### What actually happened?

The following error reported:

```
Unknown rule import-notation.stylelint(import-notation)
Unknown rule keyframe-selector-notation.stylelint(keyframe-selector-notation)
Unknown rule media-feature-range-notation.stylelint(media-feature-range-notation)
Unknown rule selector-not-notation.stylelint(selector-not-notation)
Unknown rule annotation-no-unknown.stylelint(annotation-no-unknown)
Unknown rule custom-property-no-missing-var-function.stylelint(custom-property-no-missing-var-function)
Unknown rule function-no-unknown. Did you mean unit-no-unknown?stylelint(function-no-unknown)
Unknown rule keyframe-block-no-duplicate-selectors.stylelint(keyframe-block-no-duplicate-selectors)
Unknown rule selector-anb-no-unmatchable.stylelint(selector-anb-no-unmatchable)
<element class="nuxt-logo">
[Selector Specificity](vscode-file://vscode-app/c:/Users/UserName/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-sandbox/workbench/workbench.html): (0, 1, 0)
```

### Do you have a proposal to fix the bug?

What is the problem, I need some help here

**Comment by serkon** [comment]
I have resolved upgrade package to these versions:

    "stylelint": "^15.10.3",
    "stylelint-config-recommended-vue": "^1.5.0",
    "stylelint-config-standard": "^34.0.0",

**Comment by mattxwang** [comment]
Glad it's resolved! I'm going to close this issue (since it seems unrelated to the package specifically, but rather a starter / interplay of plugins).
